charts/lenses/templates/_helper.tpl

```diff
@@ -445,6 +445,14 @@ lenses.security.ldap.plugin.memberof.key={{ .Values.lenses.security.ldap.plugin.
 lenses.security.ldap.plugin.group.extract.regex={{ .Values.lenses.security.ldap.plugin.groupExtractRegex | quote }}
 lenses.security.ldap.plugin.person.name.key={{ .Values.lenses.security.ldap.plugin.personNameKey | quote }}
 {{- end -}} 
+{{- if .Values.lenses.security.saml.enabled }}
+lenses.security.saml.base.url={{ .Values.lenses.security.saml.baseUrl | quote }}
```

Comment:
Maybe this should be `required` if saml is enabled.

```
{{ required "A valid .Values.lenses.security.saml.baseUrl is required!" .Values.lenses.security.saml.baseUrl }}
```

Reply:
then we need to do it for all the fields of saml right?